enterprise/views.py

```diff
@@ -1009,7 +1009,7 @@ class ProgramEnrollmentView(NonAtomicView):
     }
 
     @staticmethod
-    def extend_course(course):
+    def extend_course(course, enterprise_customer=None):
```

Comment:
No default value needed here.

Reply:
I added a default value because a couple of tests call this directly and don't pass in an enterprise_customer. Should I change the tests?

Reply:
Unfortunately, yes.